authentik/root/settings.py

```diff
@@ -514,3 +507,10 @@ def _update_settings(app_path: str):
 
 SHARED_APPS = list(OrderedDict.fromkeys(SHARED_APPS + TENANT_APPS))
 INSTALLED_APPS = list(OrderedDict.fromkeys(SHARED_APPS + TENANT_APPS))
```

Comment:
This should still be last, as _update_settings might update SHARED_APPS or TENANT_APPS